recipe/meta.yaml

```
@@ -23,16 +23,15 @@ requirements:
     - python >=3.7
     - pip
   run:
-    - ansiwrap
-    - black
+    - aiohttp >=3.9
```

```suggestion
    - aiohttp >=3.9,<3.10
```
Although I'm personally happy to ignore the exact pinning upstream, to be cautious we should still pin to the minor version.

IMHO pinning to a patch version is a bad anti-pattern that introduces security issues so it shouldn't be done unless you actually think there's a reasonable chance of breakage from a new patch release. The exception would be if you published a new version of your library to PyPI for each new patch version of your dependency so that people can always upgrade if security issues arise in the dependency. That's clearly not being done here so I think it's best to optimistically allow new patch versions.

---

I totally agree, thank you for the code suggestion - accepted.

---

Upstream switched to require >=3.9.0 version: https://github.com/nteract/papermill/commit/cb9fd0698b57e25b8048a5950f1e80d327f67208

Should we do the same for consistency?